Utils/Dataflow/config/009.cfg.example

```diff
@@ -32,9 +32,17 @@ mode = SQUASH
 # initial timestamp (e.g. 01-05-2016 00:00:00)
 initial = %__009_init_offset__%
 # final timestamp (e.g. 10-05-2016 00:00:00)
+# If this parameter is not set, 'now' (current moment of time) will be used.
```

Comment:
This comment is out of the commit scope, and not exactly correct: if we go "backwards" in time, it will be set to `1970-01-01 00:00:00 (UTC)`:
https://github.com/PanDAWMS/dkb/blob/715d86ff923058ea5ae9605f823b806988d709ac/Utils/Dataflow/009_oracleConnector/Oracle2JSON.py#L138-L143

Reply:
Removed the line.
While I believe that it's not out of the scope by itself (because we are adding `delay`, which has to be described, which includes the fact that it works only if `final` is not set, which invites the mentioning of stage's behavior when `final` is not set in general), you correctly point out that it works differently in a particular case. This means that this case should be described as well, and returns us to an issue already raised at some point - all the nuances of time-boundaries-related work of this stage should be documented somewhere, and doing so is definitely out of scope of the commit, and the PR.

Reply:
Added card for this issue to not forget after the conversation is resolved: https://trello.com/c/YhPuVdIk